aspnetcore/includes/not-latest-versionNM.md

```diff
@@ -8,5 +8,3 @@
 > If the selector isn't visible in a narrow browser window, widen the window or select the vertical ellipsis (**&vellip;**) > **Table of contents**.
 >
 > :::image type="content" source="~/blazor/_static/toc-selector.png" alt-text="Table of contents selector":::
```

Comment:
```suggestion
> :::image type="content" source="~/_static/toc-selector.png" alt-text="Table of contents selector":::
```

Reply:
... and there's one more there that the GH wouldn't let me submit a suggestion on.